### PR TITLE
Send a welcome email after user successfully registered

### PR DESCRIPTION
**Pull request by ko-yaroslav**

### CODE REVIEW

File: server/routes/auth/register/_post.js [hunk 1]
```diff
@@ -8,34 +10,36 @@ module.exports = async (req, res, next) => {
...
+    });
+
+    const userIsSaved = await user.save();
+
+    if (userIsSaved) {
+        const mailContent = `Hello ${firstName}. ${WELCOME_EMAIL_CONTENT}`;
+
+        sendEmail(email, 'Welcome to Worship Portal', mailContent);
+
+        res.send(userIsSaved);
```
Comment (yaroslavfedyshyn) [inline server/routes/auth/register/_post.js]: @ko-yaroslav always use res.json. This line should be out of the condition.

File: server/utils/nodeEmailer.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const transport = nodemailer.createTransport({
 
 async function sendEmail(email, subject, content) {
   const message = {
-    from: 'worship-rock@gmail.com', // Sender address
+    from: 'worship.portal.info@gmail.com', // Sender address
```
Comment (yaroslavfedyshyn) [inline server/utils/nodeEmailer.js]: @ko-yaroslav such things should be passed from env variables config file.

File: server/routes/auth/register/_post.js [hunk 3]
```diff
@@ -8,34 +10,36 @@ module.exports = async (req, res, next) => {
...
-
-        if (sameEmail) {
-          res.status(422);
-            res.json({
-                message: 'this email is already taken',
-                details: [
-                  {
+    if (sameEmail) {
+        res.status(422);
+        res.json({
```
[the remark this answers is not in the document]
Comment (ko-yaroslav) [reply server/routes/auth/register/_post.js]: Done.

File: server/routes/auth/register/_post.js (hunk 1 above)
Comment (ko-yaroslav) [reply server/routes/auth/register/_post.js]: Done.

File: server/utils/nodeEmailer.js (hunk 2 above)
Comment (ko-yaroslav) [reply server/utils/nodeEmailer.js]: Done.

**Pull request merged**